.github/CODEOWNERS

```diff
@@ -76,6 +76,9 @@
 /pkg/procmatch/                         @DataDog/burrito
 /pkg/quantile                           @DataDog/metrics-aggregation
 
+/releasenotes/                          @DataDog/agent-all
+/releasenotes-dca/                      @DataDog/agent-all
```

Comment:
for the DCA specifically, I'd leave ownership of release notes to @DataDog/container-integrations since they own the project

Reply:
Yeah that's something I was wondering about, as before the creation of the agent-platform team, this directory was owned by agent-core (by default). If you think it should belong to containers, I'll change this.